web/models/host.go

```diff
@@ -18,6 +18,18 @@ type Host struct {
 	SAPSystems    []*SAPSystem
 	AgentVersion  string
 	Tags          []string
+	CloudData     interface{}
+}
+
+type AzureCloudData struct {
+	VMName          string `json:"vmname"`
+	ResourceGroup   string `json:"resourceGroup"`
```

Comment:
Aha! `camelCase` vs `snake_case` strikes again :trollface: 

Unfortunately we are not being consistent in the way we want `json` to look like.
And I am part of the problem, too :disappointed:, for instance in #546 I have used `snake_case` 

We did it differently throughout the code and I think that at a certain point we should sit, make an agreement on the style, and then commit to refactor all the codebase accordingly, which might not be trivial.
For reference: we have also an open issue for this #385 

I don't think it is the right time to tackle such a refactor, though we can find an agreement for the immediate future.

I'd like to keep #546 consistent with this PR and since I don't have strong preferences, I can fix it to make `camelCase` compliant. Would that be ok?

Then we can have a wider discussion and consider refactoring as well.

Reply:
@nelsonkopliku @dottorblaster in the entities/models i personally have used snake case everywhere, maybe we can keep this consistent at least for this layer and think about a clean-up later. wdyt?

Reply:
Ok, whatever it works better. Anyway we'd need to find an agreement on such things. Just not now :smile:

Reply:
i like better the camel case onestly, we can refactor all at once laterz

Reply:
A bit late but ok for me.